tests/foreman/ui/test_user.py

```diff
@@ -1329,3 +1330,436 @@ def test_positive_create_in_ldap_mode(self):
                 password2='',
             )
             self.assertIsNotNone(self.user.search(user_name))
+
+
+class SshKeyInUser(UITestCase):
+    """Implements the SSH Key in User Tests"""
+
+    @stubbed
+    @tier2
+    def test_postitive_ssh_key_tab_presence(self):
+        """SSH keys tab presence in User Template
+
+        :id: a0c77cc1-0484-4290-b4b3-87ab3d0bde56
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Attempt to create new user form Super admin
+                3. Verify SSH Keys tab in user template
+
+        :expectedresults: New user template should have a tab of SSH Keys
+        """
+
+    @stubbed
+    @tier2
+    def test_postitive_ssh_key_tab_presence_Super_Admin(self):
+        """SSH keys tab presence in Super Admin Template
+
+        :id: 72dc8c6e-3627-436a-adf3-f32d09b2f1c7
+
+        :steps:
+
+                1. Go to Administer -> Users
+                2. Edit Super Admin user template
+                3. Verify SSH Keys tab in Super Admin user template
+
+        :expectedresults: Super Admin user template should have a tab of
+            SSH Keys
+        """
+
+    @stubbed
+    @tier1
+    @skip_if_bug_open('bugzilla', 1465389)
+    def test_positive_create_ssh_key(self):
+        """SSH Key can be added to User Template
```

Comment:
ssh-key can be added while creating a new user ? Please do not use template. It may confuse as we really have so many templates :)